docs/examples/movie-reviews-graph-rag.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>even though using InMemoryVectorStore is easier, we should probably use AstraDBVectorStore for all our examples to push people to use Astra.  See the CodeGeneration example for text to copy around configuration. </p><p> </p><p>Also Ben's helper mentioned above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datastax/graph-rag/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>